source/extensions/filters/http/ext_authz/ext_authz.cc

```diff
@@ -303,11 +303,8 @@ void Filter::onComplete(Filters::Common::ExtAuthz::ResponsePtr&& response) {
     }
 
     // Pass requirement for response code override as a header
-    if (config_->httpStatusForGrpcDeny()) {
-      response->headers_to_set.push_back(
-          std::pair(Http::Headers::get().ExtAuthzGrpcHttpDeny, std::string("true")));
-      ENVOY_STREAM_LOG(trace, "ext_authz filter set response-code config header",
-                       *decoder_callbacks_);
+    if (config_->retainHttpStatusOnGrpcDeny()) {
+      ENVOY_STREAM_LOG(trace, "ext_authz filter set response-code (SOON)", *decoder_callbacks_);
```

Comment:
Not totally sure we need this logging

Reply:
this is a hangover from the header-conduit approach that I missed, will remove.